public/app/screens/main/sections/devices/device-detail/style.scss

```diff
@@ -0,0 +1,97 @@
+@import "theme";
+
+// TODO: has to be a basic element
+.cc__row {
+  @include grid-row();
+  width: 100%;
+  // TODO: define right font color here
+  color: $active-gray;
+  padding: 10px 5% 0 5%;
+  &:nth-child(even) {
```

Comment:
Alternatively background color can be set for all rows and overridden for even or odd ones.